lutris/runners/residualvm.py

```diff
@@ -9,7 +9,7 @@
 
 class residualvm(Runner):
     human_name = "ResidualVM"
-    platform = "3D point-and-click games"
+    platforms = "3D point-and-click games"
```

Comment:
Grim Fandango, Escape from Monkey Isaldn and Myst 3 were relased for `('PC', 'Windows')`.